src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -228,30 +219,6 @@ private SonarQubeOrganization AskUserToSelectOrganizationOnUIThread(IEnumerable<
             });
         }
 
-        internal /*for testing purposes*/ async Task DownloadServiceParametersAsync(IProgressController controller,
```

Comment:
The only thing this step did was to pass one of the downloaded settings to the `ITestProjectRegexSetter` -> the whole method can now be deleted.